includes/wikia/models/DesignSystemSharedLinks.class.php

```diff
@@ -100,6 +95,9 @@ private function getLangWithFallback( $lang ) {
 			'wam' => 'http://de.wikia.com/WAM?langCode=de',
 			'help' => 'http://de.community.wikia.com/wiki/Hilfe:%C3%9Cbersicht',
 			'media-kit' => 'http://www.wikia.com/mediakit?uselang=de',
+			'social-facebook' => 'https://www.facebook.com/de.fandom',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "social-facebook" 11 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)